npm-shrinkwrap.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "fh-messaging-acceptance-tests",
-  "version": "0.2.2",
+  "version": "0.2.3",
   "dependencies": {
     "moment": {
-      "version": "2.15.2",
-      "from": "moment@>=2.10.6 <3.0.0",
-      "resolved": "https://registry.npmjs.org/moment/-/moment-2.15.2.tgz"
+      "version": "2.10.6",
```

Comment:
as it was "moment": "^2.10.6" it got "2.15.2" last time this was generated.
Now it's "~2.10.6", so we have downgraded the version.

We should keep the same versions we were using, and use that version with ~, not just replace `^` with `~`